tls/s2n_security_policies.c

```diff
@@ -816,6 +815,18 @@ bool s2n_ecc_is_extension_required(const struct s2n_security_policy *security_po
             return 1 == security_policy_selection[i].ecc_extension_required;
         }
     }
+
+    /* If cipher preference is not in the official list, compute the result */
+    const struct s2n_cipher_preferences *cipher_preferences = security_policy->cipher_preferences;
+    if (cipher_preferences == NULL) {
+        return false;
+    }
+    for (int j = 0; j < cipher_preferences->count; j++) {
```

Comment:
Any reason you use `j` instead of `i` here and below?

Reply:
No reason, fixed.